src/stdlib/String.ts

```diff
@@ -24,5 +24,40 @@ class LCaseCallable extends Callable {
     }
 }
 
+class AscCallable extends Callable {
+    arity = {
+        required: 1,
+        optional: 0
+    }
+
+    call(interpreter: Interpreter, str: BrsString): BrsType {
```

Comment:
Just as an FYI if you go to bash on these stdlib functions more, calling `Asc(invalid)` or `Asc(false)` probably crashes the intepreter due to the lack of runtime type checking.  It's totally my fault!  Function argument type checking is a work-in-progress, so don't panic :vulcan_salute: